trento/xml/article_sap_trento.xml

```diff
@@ -2445,6 +2446,39 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
           </step>
         </procedure>
       </section>
+
+    <section>
+      <title>Poda descriptiona and logs</title>
+      <para> In additon to the output of the Trento Support Plugin and the Dump Scenario script, the descriptions and logs of the
+       pods of the &t.server; can also be useful for analysis and troubleshooting purposes. These descriptions and logs can be 
+       easily retrieved with the <command>kubectl</command> command, for which you need a host where command <command>kubectl</command>
+       is installed and connected to the K8s cluster where Trento Server is running.
+      <procedure>
```

Comment:
I think, we miss a paragraph. In the last paragraph you describe it in a more general way. To lead the user to the procedure, I'd suggest:

```suggestion
      <para>To retrieve the logs for further introspection, do the following:
      </para>
      <procedure>
```

Reply:
Good idea adding the leading paragraph. To align better with the section goal I would write:

<para>To retrieve the descriptions and the logs for further introspection, do the following:
      </para>

Reply:
Yes, use your suggestion. :+1: